extern/CMakeProject-mad.cmake

```diff
@@ -38,6 +49,35 @@ disable_project_warnings("mad")
 
 if(MSVC)
   sm_add_compile_definition("mad" _CRT_SECURE_NO_WARNINGS)
+  sm_add_compile_definition("mad" ASO_ZEROCHECK)
+  sm_add_compile_definition("mad" $<$<CONFIG:Debug>:FPM_DEFAULT>)
```

Comment:
See [this] (https://github.com/Prcuvu/stepmania/blob/libmad_on_windows/extern/libmad-0.15.1b/msvc%2B%2B/libmad.dsp#L44) and [this line] (https://github.com/Prcuvu/stepmania/blob/libmad_on_windows/extern/libmad-0.15.1b/msvc%2B%2B/libmad.dsp#L68). I kept the difference from the original Visual C++ project.